packages/gestalt/src/Masonry.js

```diff
@@ -401,6 +401,8 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
       isVisible = true;
     }
 
+    const isRtl = this.gridWrapper ? getComputedStyle(this.gridWrapper).direction === 'rtl' : false;
```

Comment:
Curious what the render/paint performance is of calling `getComputedStyle` on every Masonry item render. https://gist.github.com/paulirish/5d52fb081b3570c81e3a#calling-getcomputedstyle

* Could/should we use `document.dir`? It's basically what we also use for RLT in our CSS: `html[dir="rtl"]` - so we don't support setting `rtl` on part for the document.
* If we need to still use `getComputedStyle`, could we do it outside of the `map` which renders every Masonry item?

Reply:
Good point. I had no idea `getComputedStyle` was so expensive — thanks for the heads-up! I'll switch to the simple `document.dir` check